src/invite/Invite.tsx

```diff
@@ -5,28 +5,44 @@ import { Share } from 'react-native'
 import { useSelector } from 'react-redux'
 import { InviteEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { INVITE_REWARDS_STABLETOKEN_LEARN_MORE } from 'src/config'
 import { inviteModal, inviteWithRewards } from 'src/images/Images'
 import { noHeader } from 'src/navigator/Headers'
 import { navigateBack } from 'src/navigator/NavigationService'
-import { inviteRewardsActiveSelector } from 'src/send/selectors'
+import { inviteRewardsActiveSelector, inviteRewardsVersionSelector } from 'src/send/selectors'
 import { useShareUrl } from './hooks'
 import InviteModal from './InviteModal'
 
 export default function Invite() {
   const { t } = useTranslation()
   const shareUrl = useShareUrl()
-  const inviteRewardsEnabled = useSelector(inviteRewardsActiveSelector)
+  const inviteRewardsActive = useSelector(inviteRewardsActiveSelector)
+  const inviteRewardsVersion = useSelector(inviteRewardsVersionSelector)
 
-  const title = inviteRewardsEnabled
-    ? t('inviteWithUrl.rewardsActive.title')
-    : t('inviteWithUrl.title')
-  const description = inviteRewardsEnabled
-    ? t('inviteWithUrl.rewardsActive.body')
-    : t('inviteWithUrl.body')
-  const image = inviteRewardsEnabled ? inviteWithRewards : inviteModal
-  const message = inviteRewardsEnabled
-    ? t('inviteWithRewards', { link: shareUrl })
-    : t('inviteWithUrl.share', { shareUrl })
+  let title = t('inviteWithUrl.title')
+  let descriptionKey = 'inviteWithUrl.body'
+  let message = t('inviteWithUrl.share', { shareUrl })
+  let image = inviteModal
+  let helpKey = ''
+  let helpLink = ''
+
+  if (inviteRewardsActive) {
+    switch (inviteRewardsVersion) {
+      case 'v4':
+        title = t('inviteWithUrl.rewardsActive.title')
+        descriptionKey = 'inviteWithUrl.rewardsActive.body'
+        message = t('inviteWithRewards', { link: shareUrl })
+        image = inviteWithRewards
+        break
+      case 'v5':
+        title = t('inviteWithUrl.rewardsActiveV5.title')
+        descriptionKey = 'inviteWithUrl.rewardsActiveV5.body'
+        message = t('inviteWithRewardsV5', { link: shareUrl })
+        helpKey = 'inviteWithUrl.help'
```

Comment:
any reason why we want this help text to only be available for v5? it seems generic enough to the claiming process, at least as far as the copy shows right now. in which case instead of the helpKey and helpLink props, we could just have a `showCollectRewardHelp` or some boolean prop

Reply:
I don't think there is any reason to limit this, I'll add the link for v4. I do think we should still have `helpLink` because different reward programs can have different links.